sktime/distances/_distance.py

```diff
@@ -694,15 +694,7 @@ def msm_distance(
     c: float = 0.0,
     **kwargs: Any,
 ) -> float:
-    """Compute the move-split-merge distance.
```

Comment:
why is this docstring removed? It was informative?

Reply:
Good spot, that's a merge error, I'll put it back